### ISSUE TITLE
Cannot Use Freeze_Graph.py and im2txt.pb of Im2Txt model

### ISSUE THREAD

**Issue by lilin29** [tensorflow/models#1163]
1.how to use a .ckpt and ckpt.meta to write to a frozen .pb with ouput_nodes_names = "softmax,lstm/initial_state,lstm/state"
2.I have tried the im2txt.pb you provided, there is an error during inference:
 Not found: FeedInputs: unable to find feed output input:0

**Comment by concretevitamin** [comment]
We'll need more information.  Could you paste platform info, the detailed commands you tried, the expected and the actual behaviors?

**Comment by karmel** [comment]
Automatically closing due to lack of recent activity. Please update the issue when new information becomes available, and we will reopen the issue. Thanks!
